### ISSUE TITLE
License

### ISSUE THREAD

**Issue by hazeycode** [Hejsil/zig-clap#64]
Sorry to raise such a dry issue...

Would it be possible to apply MIT or similar to this code?

Unlicense/Public Domain can inhibit adoption of this library since different jurisdictions have different laws pertaining to Public Domain works and relinquishment of rights over over those works.

**Comment by Hejsil** [comment]
Yea, I remember reading something about this at some point and decided that all future repos would be MIT. I wouldn't mind switching at all, but don't I have to get the :+1: from all that have contributed code to change distribution of their code under a different license?

**Comment by hazeycode** [comment]
I'm not sure about that, I'd guess you would have to but I'm not a lawyer so cannot really say

**Comment by Hejsil** [comment]
#NotALawyer
I'll look into this a little later when I get some time

**Comment by hazeycode** [comment]
:+1: thanks!

**Comment by Hejsil** [comment]
Yep, it does seem like one would need an ok from at least a majority of contributes. Ofc, the problem is always how you define things, as very few people have actually contributed new code to `zig-clap`. Most outside commits have been "update to zig master" and the like.

I'll ask anyone who have commits that are not:
* Update to newest zig
* Add X package manager module file
* Small string corrections and the like

This leaves us with @kivikakk @squeek502 and @mattnite. Not very many, but I'll ask anyway.

Relicense `zig-clap` to MIT, yay or nay?

**Comment by squeek502** [comment]
Fine with me! 

Alternatively, you could consider [the 0BSD license](https://opensource.org/licenses/0BSD) if you want to keep the 'distributing the license itself is not required' aspect of Unlicense but drop the 'relies on the concept of public domain' aspect (I've started using 0BSD over Unlicense in my more recent projects for this reason).

**Comment by kivikakk** [comment]
+1 from me. I’d be fine with 0BSD too, though I have no particular opinions about licensing.

**Comment by hazeycode** [comment]
0BSD looks great! I think I'll swap from MIT to 0BSD myself.

**Comment by mattnite** [comment]
pull the trigger

**Comment by Hejsil** [comment]
MIT it is :confetti_ball:
